File: app/src/main/java/com/happypills/db/objects/doctor.java
package com.happypills.db.objects;

import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity(tableName = "doctors")
public class doctor {

    @PrimaryKey(autoGenerate = true)
    private int uid;
    @ColumnInfo(name = "name")
    private String name;
    @ColumnInfo(name = "number")
    private String number;
    @ColumnInfo(name = "specialization")
    private String specialization;

    public int getUid() {
        return uid;
    }

    public void setUid(int uid) {
        this.uid = uid;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getNumber() {
        return number;
    }

    public void setNumber(String number) {
        this.number = number;
    }

    public String getSpecialization() {
        return specialization;
    }

    public void setSpecialization(String specialization) {
        this.specialization = specialization;
    }
}

File: app/src/main/java/com/happypills/objects/Doctor.kt
package com.happypills.objects

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "doctorsTable")
class Doctor {

    @PrimaryKey
    @ColumnInfo
    var name: String
    @ColumnInfo
    var specialization: String
    @ColumnInfo
    var phone: String

    constructor(name: String, specialization: String, phone: String) {
        this.name = name
        this.specialization = specialization
        this.phone = phone
    }
}